charts/shoot-core/components/charts/nodelocal-dns/templates/configmap.yaml

```
@@ -2,10 +2,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "node-local-dns.config.name" . }}
+  name: node-local-dns-cofigDataHash
```

```suggestion
  name: node-local-dns-configDataHash
```

Will the old ConfigMap deleted by the gardener-collector controller in GRM? Can you explain how the migration flow looks like?

---

https://github.com/gardener/gardener/pull/5125 can actually fix our issue and prevent the CM to leak. on the migration.